react/features/filmstrip/subscriber.web.js

```diff
@@ -123,3 +124,12 @@ StateListenerRegistry.register(
             );
         }
     });
+
+/**
+ * Listens for changes in the client width to determine whether the overflow menu(s) should be displayed as drawers.
+ */
+StateListenerRegistry.register(
+    /* selector */ state => state['features/base/responsive-ui'].clientWidth < 500,
+    /* listener */ (isDrawer, store) => {
```

Comment:
I blinked twice here trying to understand `isDrawer`, maybe this variable should have a name reflecting its relationship with the width?